tests/integration/golang/aim/run/get_run_metrics_test.go

```diff
@@ -0,0 +1,127 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/request"
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetRunMetricsTestSuite struct {
+	suite.Suite
+	client             *helpers.HttpClient
+	runFixtures        *fixtures.RunFixtures
+	experimentFixtures *fixtures.ExperimentFixtures
+	metricFixtures     *fixtures.MetricFixtures
+	run                *models.Run
+}
+
+func TestGetRunMetricsTestSuite(t *testing.T) {
+	suite.Run(t, new(GetRunMetricsTestSuite))
+}
+
+func (s *GetRunMetricsTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	runFixtures, err := fixtures.NewRunFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.runFixtures = runFixtures
+
+	expFixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.experimentFixtures = expFixtures
+
+	metricFixtures, err := fixtures.NewMetricFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.metricFixtures = metricFixtures
+
+	exp := &models.Experiment{
+		Name:           uuid.New().String(),
+		LifecycleStage: models.LifecycleStageActive,
+	}
+	_, err = s.experimentFixtures.CreateExperiment(context.Background(), exp)
+	assert.Nil(s.T(), err)
+
+	runs, err := s.runFixtures.CreateRuns(context.Background(), exp, 1)
+	assert.Nil(s.T(), err)
+	s.run = runs[0]
```

Comment:
Sorry, I don't understand the point when you create `array` of `runs`? Here you create single `experiment` - which is nice, but then you create array of `runs` with single run? what is the point? we also have method to create just a single run.

Reply:
the `CreateRuns` method is more convenient because it populates boilerplate, adds metrics, etc. We could refactor to separate singular/plural versions of this, it is different than the the `CreateRun` method